From d72ecaf49c520a76036aa672f150a1449ef819bc Mon Sep 17 00:00:00 2001
From: Yoni Farhadian <yonif@marvell.com>
Date: Thu, 18 Jul 2013 12:01:30 +0300
Subject: [PATCH 0845/1825] alp: remove LINUX_VERSION_CODE warning from
 checkpatch.pl

https://github.com/MISL-EBU-System-SW/misl-windriver.git linux-3.4.69-14t2-read
commit e6d5d57665efb99f14887ff86879fbb8663bccf6

	Ignore warning given by checkpatch.pl when using LINUX_VERSION_CODE
	This is needed because our drivers supports several kernel versions

Change-Id: I08838ca5b624a1f80db7845c8074c6dc68e8aa33
Signed-off-by: Yoni Farhadian <yonif@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/2717
Reviewed-by: Igor Patrik <igorp@marvell.com>
Tested-by: Igor Patrik <igorp@marvell.com>
Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 scripts/checkpatch.pl |    5 -----
 1 files changed, 0 insertions(+), 5 deletions(-)

diff --git a/scripts/checkpatch.pl b/scripts/checkpatch.pl
index 5a2dbfa..93b3de3 100755
--- a/scripts/checkpatch.pl
+++ b/scripts/checkpatch.pl
@@ -2347,11 +2347,6 @@ sub process {
 # 			$clean = 0;
 # 		}
 
-		if ($line =~ /\bLINUX_VERSION_CODE\b/) {
-			WARN("LINUX_VERSION_CODE",
-			     "LINUX_VERSION_CODE should be avoided, code should be for the version to which it is merged\n" . $herecurr);
-		}
-
 # check for uses of printk_ratelimit
 		if ($line =~ /\bprintk_ratelimit\s*\(/) {
 			WARN("PRINTK_RATELIMITED",
-- 
1.7.5.4

